canonical-rails.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency 'rails', '>= 4.1', '<= 7.0'
+  s.add_dependency 'rails', '>= 4.1', '<= 8'
```

Comment:
I think we might want to use `< 7.1` here? Or perhaps completely remove the upper dependency? wdyt @jumph4x ?

Reply:
I'll accept 7.1!